server/fileserver.go

```diff
@@ -72,3 +119,43 @@ func getUploadHandler(fsPath string) http.Handler {
 		logger.Info("Finished uploading")
 	})
 }
+
+func getDownloadHandler(fsPath string) http.Handler {
+	return http.HandlerFunc(func (res http.ResponseWriter, req *http.Request) {
+		var (
+			status int
+			err  error
+		)
+		defer func() {
+			if err != nil {
+				writeResponse(res, req, status, &Response{err.Error()})
+				return
+			}
+		}()
+		path := filepath.Join(fsPath,req.URL.String())
+		//first check if its folder or file.
+		info, err := os.Stat(path)
+		if os.IsNotExist(err) {
```

Comment:
please check if the error is not nil but doesn't suffice os.IsNotExist and handle it

Reply:
check if the error is not exist inside an if err != nil